app-emulation/gxemul/metadata.xml

```diff
@@ -1,9 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <!-- maintainer-needed -->
-  <longdescription>Machine Emulator.  Supports emulation across several architectures.  Primarily for MIPS support, but other platform support coming soon.</longdescription>
-  <upstream>
-    <remote-id type="sourceforge">gxemul</remote-id>
-  </upstream>
+	<!-- maintainer-needed -->
+	<longdescription>
+		Machine Emulator.
+		Supports emulation across several architectures.
+		Primarily for MIPS support, but other platform support coming soon.
+	</longdescription>
+	<upstream>
+		<bugs-to>mailto:gavare@gmail.com</bugs-to>
```

Comment:
Hmm I don't remember whether this required consent from the e-mail owner or not. I'll remove it in case.